apps/extension/components/iframe-popup.tsx

```diff
@@ -2,12 +2,13 @@ import { VisuallyHidden } from "@radix-ui/react-visually-hidden"
 import "@repo/tlinks/index.css"
 import { CircleX, SquareArrowUpRight } from "lucide-react"
 import { forwardRef, useImperativeHandle, useState } from "react"
+import {TokenscriptCardMetadata} from "@repo/tlinks/src/utils/fetch-ts-data.ts";
```

Comment:
The path of the import is not correct.
you can treat `@repo/tlinks` as a npm package, the `src` folder will not be included